src/libpgagroal/pipeline_session.c

```diff
@@ -592,8 +589,8 @@ session_server(struct ev_loop* loop, struct ev_io* watcher, int revents)
    client_inactive(wi->slot);
 
    exit_code = WORKER_SERVER_FAILURE;
-   running = 0;
-   ev_break(loop, EVBREAK_ALL);
+
```

Comment:
empty line, there was not a line in the previous version, keep it? This pattern repeats.